pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,12 +40,152 @@
         <Exclude>Compile</Exclude>
       </Dependency>
 
+      <!-- Only need the placeholder in the netcoreapp1.0 folder -->
       <File Include="$(PlaceholderFile)" >
         <SkipPackageFileCheck>true</SkipPackageFileCheck>
         <TargetPath>lib/netcoreapp1.0</TargetPath>
       </File>
     </ItemGroup>
   </Target>
 
+  <!--
+    Manually include the netcoreapp1.0 dependencies.
+    If any versions change in the release/1.0.0 branch they should
```

Comment:
This looks extremely fragile as it will be one more manual task to account for. Can we not skip having them?